.test-infra/jenkins/job_PerformanceTests_Python.groovy

```diff
@@ -18,46 +18,107 @@
+]
+
+
+// Configurations of each Jenkins job.
+def testConfigurations = [
+    new PerformanceTestConfigurations(
+        jobName           : 'beam_PerformanceTests_Python',
+        jobDescription    : 'Python SDK Performance Test',
+        jobTriggerPhrase  : 'Run Python Performance Test',
+        bigqueryTable     : 'beam_performance.wordcount_py_pkb_results',
+        skipPrebuild      : true,
+        pythonSdkLocation : 'build/apache-beam.tar.gz',
+        itClass           : 'apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it',
+        itModule          : 'sdks/python',
+        extraPipelineArgs : dataflowPipelineArgs + [
+            output: 'gs://temp-storage-for-end-to-end-tests/py-it-cloud/output'
+        ],
+    ),
+    new PerformanceTestConfigurations(
+        jobName           : 'beam_PerformanceTests_Python35',
+        jobDescription    : 'Python35 SDK Performance Test',
+        jobTriggerPhrase  : 'Run Python35 Performance Test',
+        bigqueryTable     : 'beam_performance.wordcount_py35_pkb_results',
+        skipPrebuild      : true,
+        pythonSdkLocation : 'test-suites/dataflow/py35/build/apache-beam.tar.gz',
+        itClass           : 'apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it',
+        itModule          : 'sdks/python/test-suites/dataflow/py35',
+        extraPipelineArgs : dataflowPipelineArgs + [
+            output: 'gs://temp-storage-for-end-to-end-tests/py-it-cloud/output'
+        ],
+    )
+]
+
+
+for (testConfig in testConfigurations) {
+  createPythonPerformanceTestJob(testConfig)
+}
+
+
+private void createPythonPerformanceTestJob(PerformanceTestConfigurations testConfig) {
+  // This job runs the Beam Python performance tests on PerfKit Benchmarker.
+  job(testConfig.jobName) {
+    // Set default Beam job properties.
+    commonJobProperties.setTopLevelMainJobProperties(delegate)
+
+    // Run job in postcommit, don't trigger every push.
+    commonJobProperties.setAutoJob(
+        delegate,
+        testConfig.buildSchedule)
+
+    // Allows triggering this build against pull requests.
+    commonJobProperties.enablePhraseTriggeringFromPullRequest(
+        delegate,
+        testConfig.jobDescription,
+        testConfig.jobTriggerPhrase)
+
+    // Helper function to join pipeline args from a map.
+    def joinPipelineArgs = { pipelineArgs ->
+      def pipelineArgList = []
+      pipelineArgs.each({
+        key, value -> pipelineArgList.add("--$key=$value")
+      })
+      return pipelineArgList.join(',')
+    }
+
+    def argMap = [
+        beam_sdk                : testConfig.sdk,
+        benchmarks              : testConfig.benchmarkName,
+        bigquery_table          : testConfig.bigqueryTable,
+        beam_it_class           : testConfig.itClass,
+        beam_it_module          : testConfig.itModule,
+        beam_prebuilt           : testConfig.skipPrebuild.toString(),
```

Comment:
the correspondence between `prebuilt` and `skipPrebuild` is not intuitive. Should the latter be called `prebuilt`?

Reply:
if the renaming makes confusion, we'd better use `prebuilt` for both.